javascript/javascript1/week1/family-pizza.js

```diff
@@ -0,0 +1,14 @@
+
+const pizzasPrice = 70;
+let amountOfPizzas = 3;
+let pizzaSize = 'Family Pizza';
+
+if (pizzaSize)
+{
+    totalPrice = 2 * pizzasPrice;//Family pizza price is double the price of normal pizza
```

Comment:
This works but you should still declare the variable `totalPrice` up top. It makes the code clearer.